src/components/Modals/ShareAdsModal/ShareAdsModal.tsx

```diff
@@ -65,6 +64,7 @@ const ShareAdsModal = ({ id, isModalOpen, onRequestClose }: TShareAdsModalProps)
         if (divRef.current) {
             const p2pLogo = divRef.current.querySelector('.share-ads-card__qr-icon');
             if (p2pLogo) {
+                const { default: html2canvas } = await import('html2canvas');
```

Comment:
Tested locally for `html2canvas` export:

https://github.com/deriv-com/p2p/assets/103016120/6fb19e7a-0202-4a1b-8c68-cdc5c5c76608